File: src/main/java/com/example/learninghub/Controller.java
package com.example.learninghub;

import org.springframework.web.bind.annotation.CrossOrigin;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class Controller {
    @GetMapping("/")
    public String get() {
        System.out.println("Default call running");
        return "I'm server!";
    }

    @CrossOrigin(origins = "https://learning-hub-ui.vercel.app")
    @GetMapping("/v1/health-check")
    public String health() {
        System.out.println("Health-check running");
        return "Healthy";
    }
}

File: src/main/kotlin/com/learning/hub/Controller.kt
package com.learning.hub

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class Controller {
    @GetMapping("/")
    fun default() = "I'm test server"

    @GetMapping("/v1/health-check")
    fun get() = "Server is healthy"
}